scipy/special/_generate_pyx.py

```diff
@@ -316,8 +316,10 @@ def generate_loop(func_inputs, func_outputs, func_retval,
     else:
         rv = ""
 
-    funcall = "        {}(<{}(*)({}) noexcept nogil>func)({})\n".format(
-        rv, CY_TYPES[func_retval], ", ".join(ftypes), ", ".join(fvars))
+    funcall = (
+    f"        {rv}(<{CY_TYPES[func_retval]}(*)({', '.join(ftypes)}) "
+    f"noexcept nogil>func)({', '.join(fvars)})\n"
+    )
 
```

Comment:
This is one more change that I think needs a close look. I'm assuming the spacing is important for generating the pyx file with the right indentation. I believe I got it right, but please check.

Reply:
this all looks good to me 👍